diff --git a/node_modules/@xmpp/resolve/resolve.js b/node_modules/@xmpp/resolve/resolve.js
index 02d38e2..526e284 100644
--- a/node_modules/@xmpp/resolve/resolve.js
+++ b/node_modules/@xmpp/resolve/resolve.js
@@ -3,7 +3,7 @@ import * as http from "./lib/http.js";
 
 export default function resolve(...args) {
   return Promise.all([
-    dns.resolve ? dns.resolve(...args) : Promise.resolve([]),
+    dns?.resolve?.(...args) ?? [],
     http.resolve(...args),
   ]).then(([records, endpoints]) => [...records, ...endpoints]);
 }
